doc/specs/vulkan/appendices/VK_EXT_direct_mode_display.txt

```diff
@@ -37,16 +37,18 @@ None.
 === Issues
 
 1) Should this extension and its related platform-specific extensions
-leverage +VK_KHR_display+, or provide separate equivalent interfaces.
-
-*RESOLVED*: Use +VK_KHR_display+ concepts and objects.
-+VK_KHR_display+ can be used to enumerate all displays on the system,
-including those attached to/in use by a window system or native platform,
-but +VK_KHR_display_swapchain+ will fail to create a swapchain on in-use
-displays.
+leverage <<VK_KHR_display,`VK_KHR_display`>>, or provide separate equivalent
```

Comment:
AFAICT it's fine to use the more compact markup - seems to do the right thing in both HTML and PDF outputs.  If you want to update the MR soon along these lines that would be good, otherwise when I accept it I'll make corresponding edits myself.

> `<<VK_KHR_display>>`

Reply:
on it